### PR TITLE
Work on the Gemfile to make bundle update pass

### PR DESCRIPTION
**Pull request by sauloperez**
#### What? Why?

This comments out a number of gems that either are fetch from out-of-date forks, that are meant for Rails 3 only or that do not support Rails 4.

There a lot of work still to be done to have a working Gemfile.

#### How is this related to the Spree upgrade?

It is the very first step towards v2.1. What is supposed to be the next upgrade we do after v2.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,16 +21,16 @@ gem 'pg'
...
 # Our branch contains two changes
 # - Pass customer email and phone number to PayPal (merged to upstream master)
 # - Change type of password from string to password to hide it in the form
-gem 'spree_paypal_express', github: "openfoodfoundation/better_spree_paypal_express", branch: "2-0-stable"
+# gem 'spree_paypal_express', github: "openfoodfoundation/better_spree_paypal_express", branch: "2-0-stable"
 gem 'stripe', '~> 4.5.0'
 # We need at least this version to have Digicert's root certificate
 # which is needed for Pin Payments (and possibly others).
-gem 'activemerchant', '~> 1.78'
+# gem 'activemerchant', '~> 1.78'
```
Comment (sauloperez) [inline Gemfile]: If this particular version is not compatible with Spree we will need to first understand what is it that we need from activemerchant (I don't remember) and possibly bring in that Digicert's root certificate from somewhere else.

File: Gemfile [hunk 2]
```diff
@@ -59,7 +58,7 @@ gem 'rabl'
 # Once Rails is updated to 5.x we should bump directly to 0.10.x
 gem "active_model_serializers", "0.8.4"
 gem 'oj'
-gem 'deface', '1.0.0'
+gem 'deface'
```
Comment (sauloperez) [inline Gemfile]: I even wonder if we need to specify deface ourselves. It is Spree itself that depends on it on a particular version.

**Comment by sauloperez** [comment]
What should we do with this @luisramos0 ? Feel free to assign yourself if you want work on it. I unassign myself.

**Comment by luisramos0** [comment]
this PR can be closed as I have put all this in 2-1-stable already.

**Comment by luisramos0** [comment]
I am blocked here: https://github.com/openfoodfoundation/openfoodnetwork/issues/3565#issuecomment-475419217

can you have a quick look, maybe you will understand the errors in the build and why I dont see them locally?

**Pull request closed without merging**